common/changes/office-ui-fabric-react/colorPickerPerf2_2019-06-10-20-51.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Removing getClassNames method from SwatchColorPicker.base.tsx",
```

Comment:
You might consider noting in the comment what the outcome of the change is not necessary what changed in implementation. The audience consists of dependents of the library. Noting a performance fix would likely get folks attention :).

Reply:
Yup...done. Thanks !